### PR TITLE
task-6 Viktor Oleshkevich

### PR DESCRIPTION
**Pull request by BlackHatMan**
https://travis-ci.org/github/BlackHatMan/js-assignments/builds/723428197

### CODE REVIEW

File: task/06-conditions-n-loops-tasks.js [hunk 1]
```diff
@@ -30,7 +30,13 @@
  *
  */
 function getFizzBuzz(num) {
-    throw new Error('Not implemented');
+
+    if(num %5 == 0 & num %3 ==0)return 'FizzBuzz' ;
+    else if(num %3 ==0)         return 'Fizz';  
+    else if(num %5 == 0)       return 'Buzz' ;
```
Comment (vramaniuk) [inline task/06-conditions-n-loops-tasks.js]: "operator === is always preferable
sometimes ''==' may cause an error
(check at console :
""0"" == 0
and ""0"" === 0)"

File: task/06-conditions-n-loops-tasks.js [hunk 2]
```diff
@@ -209,7 +245,12 @@ function getIntervalString(a, b, isStartIncluded, isEndIncluded) {
  * 'noon' => 'noon'
  */
 function reverseString(str) {
-    throw new Error('Not implemented');
+    let buffer= '';
+    for( let i = str.length -1; i>=0; i--)
+    {
+    buffer +=str[i];
+    }
+    return buffer;
```
Comment (vramaniuk) [inline task/06-conditions-n-loops-tasks.js]: Please, try to solve this task using Array method 'reverse'
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reverse
We can transform string into array  at first.

File: task/06-conditions-n-loops-tasks.js [hunk 3]
```diff
@@ -226,7 +267,14 @@ function reverseString(str) {
...
-    throw new Error('Not implemented');
+    let reverse=[];
+    let buffer = num.toString();
+    for( let i = buffer.length -1; i>=0; i--)
+    {
+        reverse +=buffer[i];
+    }
+    return reverse;
+
 }
```
Comment (vramaniuk) [inline task/06-conditions-n-loops-tasks.js]: Please, try to solve this task using Array method 'reverse'
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reverse
We can convert number  into string  at first, then transform it into array.

**Comment by vramaniuk** [comment]
> https://travis-ci.com/github/BlackHatMan/js-assignments

Please, privide the direct link to your task success travis build
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by vramaniuk** [comment]
Why did you add 07-yield-tasks solution into this PR(task-6-branch)? commit 7b04542

**Comment by BlackHatMan** [comment]
> Why did you add 07-yield-tasks solution into this PR(task-6-branch)? commit [7b04542](https://github.com/AisBrestEDU/js-assignments/commit/7b0454232e86c03d72c0fe9a8034bc791b6d412f)
seems to have fixed))

**Comment by vramaniuk** [comment]
Closing the PR as accepted! Thanks!

**Pull request closed without merging**